### ISSUE TITLE
direction issue 

### ISSUE THREAD

**Issue by shabane** [gethugothemes/bookworm-light#12]
hello again.
this theme is the **best theme** ever.
but it have some issue for right to left language's like Persion(Farsi).
in the other hugo them like **cupper** we used to add a `<div dir'rtl'>` at the top of each paragraph and a `</div>` at the end to show text in the **right** instead of left. but in this theme this trick doesn't work.

### code
![Screenshot from 2021-12-24 14-26-40](https://user-images.githubusercontent.com/51704066/147347617-47b51497-6e51-4b2d-82d3-8bd8823bd95d.png)

### preview
![rtl-issue](https://user-images.githubusercontent.com/51704066/147349626-62aea796-7e14-4684-95b6-e58e86c2f6c1.jpg)

**Comment by shabane** [comment]
@Gnkalk had a persion theme, my he can help us

**Comment by Gnkalk** [comment]
You can use https://github.com/FOSSFA/bookworm-FE
I tried to translate this template as well as I could, but a lot of work doesn't allow me to complete the translation process, you can take a fork of it and complete it.

@shabane @tfsomrat

**Comment by davidsneighbour** [comment]
In my opinion, the theme needs to be adapted to Bootstrap 5, which has RTL support integrated. Some work needs to be done exchanging `left` and `right` padding/margin/floating etc to `start` and `end`. After that is should be as easy as flipping a switch to change a layout from rtl to ltr and back.

**Comment by shabane** [comment]
> You can use https://github.com/FOSSFA/bookworm-FE

yes i know your beautiful theme, but i liked this more than the others.
i just mention you because i thought may you have some idea to help us for this issue

**Comment by Gnkalk** [comment]
> > You can use https://github.com/FOSSFA/bookworm-FE
> 
> yes i know your beautiful theme, but i liked this more than the others. i just mention you because i thought may you have some idea to help us for this issue

I have two recommendations :

1.  Menus , Navbar and others be reversed
2.  theme search engine for Persian and Arabic and other RTL languages is not well optimized and does not provide the correct result.
